docs/src/language_servers.md

```diff
@@ -16,6 +16,28 @@ If there are no setup steps for a language server on this page, but a [language
 
 Follow installation instructions on [LSP-angular](https://github.com/sublimelsp/LSP-angular).
 
+## Assembly
+
+1. Install `asm-lsp` via Cargo (see [github:bergercookie/asm-lsp](https://github.com/bergercookie/asm-lsp)):
+
+    ```sh
+    cargo install asm-lsp
+    ```
+
+2. Open `Preferences > Package Settings > LSP > Settings` and add the `"asm-lsp"` client configuration to the `"clients"`:
+
+    ```jsonc
+    {
+        "clients": {
+            "asm-lsp": {
+                "enabled": true,
+                "command": ["asm-lsp"],
+                "selector": "source.asm | source.assembly"
+            }
+        }
+    }
+    ```
```

Comment:
You should also specify which syntax should be installed since the server won't start without it (since the scope won't match).